src/boards/include/boards/waveshare_rp2040_lcd_0.96.h

```diff
@@ -31,13 +31,13 @@
 
 // --- I2C ---
 #ifndef PICO_DEFAULT_I2C
-#define PICO_DEFAULT_I2C 0
+#define PICO_DEFAULT_I2C 1
 #endif
 #ifndef PICO_DEFAULT_I2C_SDA_PIN
-#define PICO_DEFAULT_I2C_SDA_PIN 4
+#define PICO_DEFAULT_I2C_SDA_PIN 6
```

Comment:
How come this has changed? Was the original _always_ wrong, or are there now two separate revisions of this board?

Reply:
The board is not modified, and it is not an wrong, but Waveshare's extension board usually uses I2C pins are GPIO6 and GPIO7, modify the I2C description here to help users use